core/index.js

```diff
@@ -281,9 +282,15 @@ DattCore.prototype.asyncNewContentAuth = function (title, label, body) {
  */
 DattCore.prototype.asyncPostContentAuth = function (contentauth) {
   return asink(function *() {
-    let msg = MsgContentAuth().fromContentAuth(contentauth).toMsg()
-    this.broadcastMsg(msg)
-    return this.corecontent.asyncPostContentAuth(contentauth)
+    let msgauth = MsgContentAuth().fromContentAuth(contentauth)
+    return this.asyncPostMsgAuth(msgauth)
+  }, this)
+}
+
+DattCore.prototype.asyncPostMsgAuth = function (msgauth) {
```

Comment:
This method should have at least one test. Notice how the other tests in dattcore mock up things like broadcastMsg.